.default.env

```diff
@@ -1,2 +1,3 @@
 ENVIRONMENT=development
 SWARM_EXTENSION_ID=oncdlaccbflglnchidicplbcikejjcid
+POSTAGE_BATCH_ID=38ebc05cecd2451a5dd57fd26eed593e84a2b917e3aa239d2c288331c61b59f8
```

Comment:
is this postage batch id can be with full of zeros? because it should be handled by either swarm-extension or blossom